pcdsdevices/areadetector/detectors.py

```diff
@@ -63,6 +64,28 @@ def get_plugin_graph_edges(self, *, use_names=True, include_cam=False):
                           for src, dest in port_edges]
         return port_edges
 
+    def screen(self, main=False):
+        """
+        Open camViewer screen for camera.
+
+        Parameters
+        ----------
+        main : bool, optional
+            Set to True to bring up 'main' edm config screen.
+            Defaults to False, which opens python viewer.
+        """
+        if shutil.which('camViewer'):
+            arglist = ['camViewer', '-H', str(get_hutch_name()).lower(),
+                       '-c', self.name]
+            if main:
+                arglist.append('-m')
+
+            logger.info('starting camviewer')
```

Comment:
```suggestion
            self.log.info('starting camviewer')
```
The per-Device logger has metadata including the object name, so it's preferable to use it

Reply:
I wanted to make sure that the logging info printed to the terminal, since I'm making these methods as quiet as possible otherwise.  Do we expect the per-Device logger to also print to the hutch-python session?

Reply:
I would have expected the per-device logger to show in the hutch-python session. Otherwise we have some sort of bug.